### ISSUE TITLE
[Accessibility]: Hint text does not pass contrast ratios on non-white background

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/vets-design-system-documentation#5082]
### Point of contact

Laura Flannery

### Defect Severity

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Severity rationale

This issue is occurring on the homepage of VA.gov. Users with low vision, color blindness or other vision issues may not be able to see this text at all

### URL where issue occurs

https://www.va.gov/

### Component or pattern (if applicable)

va-text-input - the hint text color (in this case being used for the character counter)

### Environment details

Not environment specific

### Steps to reproduce

1. Go to the VA.gov homepage
2. Scroll to the bottom of the page, to the Email section
3. Notice the character counter
4. Run axeDev tools
5. See the color contrast violation that this hint text color does not pass color contrast ratio requirements against the blue background. 

(I also know that this color doesn't pass against a light gray background either.)

### Supporting evidence (screenshots, recordings, code snippets)

<img width="1144" height="643" alt="Image" src="https://github.com/user-attachments/assets/b063c89e-8175-4451-bc54-07e891a8d6fc" />

### Type of issue

- [x] Axe-core
- [ ] Cognitive | user flows & navigating
- [x] Color, typography, & visual elements
- [x] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [ ] Focus
- [ ] Headings
- [ ] Keyboard
- [ ] Link and button labels
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Screenreader | assistive tech & device support
- [ ] Semantic HTML
- [ ] URL standards
- [ ] Zoom or Magnification
- [ ] Something else

**Comment by amyleadem** [comment]
Confirming that the current color combination of `#71767A` (` uswds-system-color-gray-cool-50`) and `#e1f3f8` (`uswds-system-color-blue-cool-vivid-5`) have a contrast ratio of 4.01:1. To meet contrast ratio requirements on this background color, the hint text would need to have a USWDS color grade of at least 60. 

Some possible solutions: consider changing all hint text to grade 60, create new color rules for hint text that work in combination with our background color utilities, or open the door for teams to customize styles for the hint text color.

**Comment by tbaker1026** [comment]
This is no longer an issue - closing issue
